app/models/vim_performance_daily.rb

```diff
@@ -4,22 +4,6 @@ def self.instances_are_derived?; true; end
   INFO_COLS = [:resource_type, :resource_id, :resource_name]
   PARENT_COLS = [:parent_host_id, :parent_ems_cluster_id, :parent_storage_id, :parent_ems_id].freeze
 
-  def self.find(cnt, *args)
-    raise _("Unsupported finder value %{number}") % {:number => cnt} unless cnt == :all
-    Vmdb::Deprecation.deprecation_warning(:find, :find_entries, caller(1))
-
-    all(*args)
-  end
-
-  def self.all(*args)
-    Vmdb::Deprecation.deprecation_warning(:all, :find_entries, caller(1))
-    options = args.last.kind_of?(Hash) ? args.last : {}
-    ext_options = options.delete(:ext_options) || {}
-    find_entries(ext_options)
-      .apply_legacy_finder_options(options)
-      .to_a
-  end
-
```

Comment:
@Fryguy yes please
Thought I already had an outstanding pr for this.

Burn this method with 🔥